.github/workflows/build.yaml

```diff
@@ -130,7 +130,9 @@ jobs:
           - nightly-2022-07-22
     steps:
       - uses: actions/checkout@v3
-      - uses: mobilecoinfoundation/actions/sgxsdk@main
+      - uses: mobilecoinfoundation/actions/sgxsdk@sgx_sdk_2_18
```

Comment:
More a note to myself, but this branch probably shouldn't be merged in until https://github.com/mobilecoinfoundation/actions/pull/2 is so that we can go back to `@main`